### PR TITLE
buyer is able to add items to cart

### PR DESCRIPTION
**Pull request by Ishimwe7**
### What does this PR do?
This PR implements the functionality that allows buyers to add items to their shopping cart.
### Description of Task to be completed?
Create a 'Add to Cart' button on the product detail and listing pages.
Implement the functionality to add selected items to the user's cart.
### How should this be manually tested?
1.Login first
2. Navigate to a product listing or detail page.
3.Click on  a single  product. you want to view full information
4. Click add to cart button.

### What are the relevant pivotal tracker stories
https://www.pivotaltracker.com/story/show/187570834

### CODE REVIEW

**Review by Balinda21 — APPROVED** [review]
We are good to go,
well done @Ishimwe7

**Review by rodriguecyber — APPROVED** [review]

File: src/pages/Cart.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
```
Comment (kirengamartial) [inline src/pages/Cart.tsx:1]: this file is empty if your are not using it you may remove it

File: src/slices/cartSlice/cartSlice.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+//   items: [],
+//   status: 'idle',
+//   error: null,
+// };
+
+// export const addToCartAsync = createAsyncThunk(
+//   'cart/addToCart',
+//   async ({ productId, quantity }: { productId: number; quantity: number }) => {
+//     const response = await fetch('/cart/add', {
+//       method: 'POST',
```
Comment (kirengamartial) [inline src/slices/cartSlice/cartSlice.tsx]: remove this comments they are too many

File: vite.config.ts [hunk 3]
```diff
@@ -1,3 +1,25 @@
+// import { defineConfig } from "vite";
+// import react from "@vitejs/plugin-react";
```
Comment (kirengamartial) [inline vite.config.ts:2]: even this comments remove them

**Review by kirengamartial — CHANGES_REQUESTED** [review]
remove yarn from being tracked by git

File: yarn.lock
Comment (robsdagreat) [inline yarn.lock:1]: You forgot to ignore this file.

**Review by robsdagreat — COMMENTED** [review]
Everything looks good except for the comments we left for you. So consider working on them.

**Review by DeliceLydia — APPROVED** [review]

**Review by kirengamartial — APPROVED** [review]

**Review by robsdagreat — APPROVED** [review]

**Pull request merged**